kyuubi-server/pom.xml

```diff
@@ -395,6 +395,23 @@
             <artifactId>swagger-ui</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
```

Comment:
should be provided or optional, avoid to  propagate to the engine moudles

Reply:
SGTM.